### PR TITLE
docs: Update readme etc

### PR DESCRIPTION
**Pull request by geoffturk**
[PR Process](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-process) - [PR Review Checklist](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-review-checklist)

### Release

Semantic release is enabled for this repository. Make sure you follow the right commit message convention. 

We're using semantic-release's default — [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).

## Description of Changes

Edits to README, plus cleanup of unit test boilerplate and minor updates to package.json (change description, add `npm test` to run unit tests, move @types to dependencies). Added some hidden TODOs in README as reminders for future improvements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,34 +86,26 @@ const response = await fetch(poetNodeUrl + '/identities/', {
...
 
 ### Coverage
-Coverage is generated with [Istanbul](https://github.com/istanbuljs/nyc) whenever tests are run.
-A more complete report can be generated by running `npm run coverage`.
 
-This area needs some reviewing — the reports look a bit off sometimes.
+Coverage reports are created with [Istanbul](https://github.com/istanbuljs/nyc), which can be generated by running `npm run coverage`.
 
-Also, we'll want the Travis job to check that coverage doesn't go down with pull requests. 
+<!-- TODO: This area needs some reviewing; the reports look a bit off sometimes. Also, we'll want the Travis job to check that coverage doesn't go down with pull requests. -->
```
Comment (lautarodragan) [inline README.md]: @warrenv @wzalazar maybe we can now complete remove this?

File: package.json [hunk 2]
```diff
@@ -35,6 +36,7 @@
   },
   "dependencies": {
     "@types/bitcore-lib": "0.15.1",
+    "@types/node-fetch": "1.6.9",
```
Comment (lautarodragan) [inline package.json]: I think they are needed as normal dependencies due to Po.et JS being a library, not 100% sure now though. Will try to check that later today — feel free to leave it out of this PR to get it merged faster though.

Comment (geoffturk) [reply package.json]: Check out this discussion: https://github.com/Microsoft/types-publisher/issues/81

This is what persuaded me to move them into `dependencies`.

File: README.md (hunk 1 above)
Comment (geoffturk) [reply README.md]: I will remove it. @warrenv or @wzalazar can create a ticket for it if it is still an issue.

File: package.json (hunk 2 above)
Comment (geoffturk) [reply package.json]: I've gone through the thread again and looked at a couple of other discussions on this topic and the consensus is that it should be in dependencies if the package is going to be a library. See here for another example:
https://github.com/apollographql/apollo-client/issues/713

I'm pretty convinced this is the right path.

Comment (geoffturk) [reply package.json]: I don't think the size of the @type packages is an issue because they are each 16kb uncompressed, so hardly worth worrying about.

**Pull request merged**